vscode-lean4/src/utils/projectInfo.ts

```diff
@@ -99,63 +103,38 @@ export async function findLeanPackageVersionInfo(uri: Uri) : Promise<[Uri | null
 
 // Find the 'leanpkg.toml' or 'lean-toolchain' in the given package root and
 // extract the Lean version info from it.
-export async function readLeanVersion(packageUri: Uri) : Promise<string | null> {
+export async function readLeanVersion(packageUri: Uri) : Promise<string> {
     const toolchainFileName = 'lean-toolchain';
     const tomlFileName = 'leanpkg.toml';
     if (packageUri.scheme === 'file') {
         const leanToolchain = Uri.joinPath(packageUri, toolchainFileName);
-        if (fs.existsSync(new URL(leanToolchain.toString()))) {
+        try {
             return await readLeanVersionFile(leanToolchain);
-        }
-        else {
-            const leanPkg = Uri.joinPath(packageUri, tomlFileName);
```

Comment:
This might be a bit subtle, but we're actually trying to read two different files here: `lean-toolchain` and `leanpkg.toml`.  You've just deleted the `leanpkg.toml` part, please put it back.

Reply:
Sure! thanks guys!! :)